llvm/include/llvm/IR/Type.h

```diff
@@ -460,6 +463,7 @@ class Type {
   static IntegerType *getInt1Ty(LLVMContext &C);
   static IntegerType *getInt8Ty(LLVMContext &C);
   static IntegerType *getInt16Ty(LLVMContext &C);
+  static IntegerType *getInt20Ty(LLVMContext &C);
```

Comment:
Unless this is going to be used by some generic passes, I would keep that in some utils in your backend.